universal-login-react/src/ui/Notifications/ConnectionNotification.tsx

```diff
@@ -50,23 +55,19 @@ export const ConnectionNotification = ({deployedWallet, devicesBasePath, classNa
     <div id="notifications" className="universal-login-emojis">
       <div className={getStyleForTopLevelComponent(className)}>
         <div className="approve-device">
-          {notifications.length > 0 && (
+          {showTitle &&
```

Comment:
Maybe we should think about changing name because this variable defines if we show the whole text not the only title.

Reply:
`showHeader` maybe, the header would be title+text.
I'd say this rename is optional in this PR.